docs/Versions.md

```diff
@@ -11,6 +11,7 @@ Versioned formulae we include in [homebrew/core](https://github.com/homebrew/hom
 * Versioned formulae should differ in major/minor (not patch) versions from the current stable release. This is because patch versions indicate bug or security updates and we want to ensure you apply security updates.
 * Unstable versions (alpha, beta, development versions) are not acceptable for versioned (or unversioned) formulae.
 * Upstream should have a release branch for the versioned formulae version and still make security updates for that version, when necessary. For example, [PHP 5.5 was not a supported version but PHP 7.2 was](https://php.net/supported-versions.php) in January 2018.
+* Versioned formula should share a codebase with the main version. If the project is split into a different repository a new formula should be made.
```

Comment:
Not sure I agree with this. What's your reasoning/motivation?

Reply:
As far as I could tell this is just a writeup of the way this is currently handled in core. `should be` might be a bit much here though. What I was getting at was the difference between a new branch (should be `formula@version`) and a new repo continuing the work of a tool (should be `formulaversion`).

Reply:
Gotcha. How about then:

```suggestion
* Versioned formulae share a codebase with the main formula. If the project is split into a different repository we recommend creating a new formula (`formula2` rather than `formula@2` or `formula@1`).
```

Reply:
That's better, thanks for doing meh words gooder